diff --git a/gi/t-/git-shell b/gi/t-/git-shell
index 07587b7b5e..2caa833264 100644
--- a/gi/t-/git-shell
+++ b/gi/t-/git-shell
@@ -8,3 +8,4 @@
 {"name":"git-shell","vers":"0.1.7+main","deps":[{"name":"dia-semver","req":">=9, <10","features":[],"optional":false,"default_features":true,"target":null,"kind":"normal"}],"cksum":"8a495f2c425e6cb8e627cd0980fa99f37c8914034b29b52bce1dfaa14464fd45","features":{},"yanked":false,"links":null}
 {"name":"git-shell","vers":"0.2.0","deps":[{"name":"dia-semver","req":">=9.0.1, <10","features":[],"optional":false,"default_features":true,"target":null,"kind":"normal"}],"cksum":"7f5780bd30eb5d5e0e16ec655ea79d9a29371e64e59584248b8fadc3bfdbe074","features":{},"yanked":false,"links":null}
 {"name":"git-shell","vers":"0.3.0","deps":[{"name":"dia-semver","req":">=9.0.1, <10","features":[],"optional":false,"default_features":true,"target":null,"kind":"normal"}],"cksum":"57f518aa81cbeb005305e8d231d7c0cceaeb3960ebce7c81cb034f5ac0585290","features":{},"yanked":false,"links":null}
+{"name":"git-shell","vers":"0.3.1","deps":[{"name":"dia-semver","req":">=9.0.1, <10","features":[],"optional":false,"default_features":true,"target":null,"kind":"normal"}],"cksum":"627e33558be3c15d39f9026336765c307f129314ef06e597b131e4a0953ce1dc","features":{},"yanked":false,"links":null}
