diff --git a/gi/t-/git-status-vars b/gi/t-/git-status-vars
new file mode 100644
index 0000000000..17dcbe88f1
--- /dev/null
+++ b/gi/t-/git-status-vars
@@ -0,0 +1 @@
+{"name":"git-status-vars","vers":"0.1.1","deps":[{"name":"clap","req":"^3.2.15","features":["derive"],"optional":false,"default_features":true,"target":null,"kind":"normal"},{"name":"git2","req":"^0.15.0","features":[],"optional":false,"default_features":false,"target":null,"kind":"normal"},{"name":"shell-words","req":"^1.1.0","features":[],"optional":false,"default_features":true,"target":null,"kind":"normal"},{"name":"tempfile","req":"^3.3.0","features":[],"optional":false,"default_features":true,"target":null,"kind":"dev"}],"cksum":"bfdb16b742c4753d19e5177bd9d87ecd8ab8ab462aaad245af44d4bb4261a89f","features":{},"yanked":false,"links":null}
