diff --git a/gi/t_/git_function_history b/gi/t_/git_function_history
index 56aa1e5a63..7abcc7ecad 100644
--- a/gi/t_/git_function_history
+++ b/gi/t_/git_function_history
@@ -1 +1,2 @@
 {"name":"git_function_history","vers":"0.1.0","deps":[],"cksum":"0c89d211d4ca387f270aa81cc2b78356b9570865074f3c13c688e0206423f945","features":{},"yanked":false,"links":null}
+{"name":"git_function_history","vers":"0.1.1","deps":[],"cksum":"db46181a6a6a8b18c7603942b827514153ca4479a6c47b61d520d5fe6a2938dd","features":{},"yanked":false,"links":null}
